doc/hardware/porting/board_porting.rst

```diff
@@ -65,112 +65,128 @@ complete conversion reference.
-use: ``plank/soc1/foo``, and if omitting the SoC use: ``plank//foo``.
-Here the double ``//`` indicates to the build system that the SoC has been
-omitted.
+   +--------------------------------------------+--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   | :term:`board name`                         | :term:`board qualifiers` | :term:`SoC` | :term:`SoC Series` | :term:`SoC family` | CPU core       | :term:`architecture` |
+   +============================================+==========================+=============+====================+====================+================+======================+
+   | :ref:`nrf52dk <nrf52dk_nrf52832>`          | nrf52832                 | nRF52832    | nRF52              | Nordic nRF         | Arm Cortex-M4  | Arm Cortex-M         |
+   +--------------------------------------------+--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   | :ref:`frdm_k64f <frdm_k64f>`               | mk64f12                  | MK64F12     | Kinetis K6x        | NXP Kinetis        | Arm Cortex-M4  | Arm Cortex-M         |
+   +--------------------------------------------+--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   | :ref:`rv32m1_vega <rv32m1_vega>`           | openisa_rv32m1/ri5cy     | RV32M1      | (Not used)         | (Not used)         | RI5CY          | RISC-V               |
+   +--------------------------------------------+--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   | :ref:`nrf5340dk <nrf5340dk_nrf5340>`       | nrf5340/cpuapp           | nRF5340     | nRF53              | Nordic nRF         | Arm Cortex-M33 | Arm Cortex-M         |
+   |                                            +--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   |                                            | nrf5340/cpunet           | nRF5340     | nRF53              | Nordic nRF         | Arm Cortex-M33 | Arm Cortex-M         |
+   +--------------------------------------------+--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   | :ref:`mimx8mp_evk <imx8mp_evk>`            | mimx8ml8/a53             | i.MX8M Plus | i.MX8M             | NXP i.MX           | Arm Cortex-A53 | Arm64                |
+   |                                            +--------------------------+-------------+--------------------+--------------------+----------------+----------------------+
+   |                                            | mimx8ml8/m7              | i.MX8M Plus | i.MX8M             | NXP i.MX           | Arm Cortex-M7  | Arm Cortex-M         |
```

Comment:
architecture isn't really right, M4 would be armv7, M33 would be armv8

Reply:
yes, good point. I will fix that.